### DISCUSSION TITLE
How to disable syntax?

### DISCUSSION THREAD

**Discussion by aweikalee** [Milkdown/milkdown#1550] [Q&A]
I only need some of the markdown syntax and want to disable unused syntax.
I tried to remove some schema, but get error "Cannot match target parser for node:".

**Comment by Saul-Mirone** [comment]
You need to override the schemas you don't need. Change to `toDom` and `fromDom` to something like empty nodes. But it's not recommended. A better way is transform and filter your markdown source file before it is parsed by milkdown.

**Reply by kessiler** [reply]
@Saul-Mirone would you kindly give an example?
